sema/discovery/discovery.py

```
@@ -170,22 +213,26 @@
-                # if so then add it to the triplestore
-                for script_type, script_content in script.items():
-                    if script_type in self.SUPPORTED_MIMETYPES:
-                        self._add_triples_from_text(
-                            script_content, script_type, resp.url
-                        )
-            parser.close()
+            l_s_dict = self._parse_url_content(resp.text, resp.url)
+
+            if l_s_dict["links"] == 0 and l_s_dict["scripts"] == 0:
+                log.info(
+                    f"no links or scripts found in the html content of {resp.url}"
```

Ruff

Logging statement uses f-string

(G004)